app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2> User Summary: <%= @user.name %> </h2>
+<sub> Joined site <%= @user.created_at %> </sub>
+
+
+<h3> Votes </h3>
+<table class="table">
+
+  <tr>
+    <th>Media Title</th>
+    <th>Created by</th>
+    <th>Published</th>
+    <th>Category</th>
+    <th>Voted on</th>
+  </tr>
+
+<% @user.votes.each do |vote| %>
+
+  <tr>
+    <td> <%= link_to vote.work.title, work_path(vote.work.id) %> </td>
```

Comment:
If `vote.work` is `nil` (aka if the work was deleted, you have a relationship to nullify the work off the vote), then `vote.work.title` will break the site